Editor/EditorCore.cs

```diff
@@ -369,6 +369,118 @@ private static void GetSceneVersionResponse(int responsecode, string error, stri
         }
 
 #region GUI
+        internal class Styles
```

Comment:
Brought this style into EditorCore so it can be used globally and not just in project validation